API.md

```diff
@@ -0,0 +1,203 @@
+# CodeReady Analytics Core API Specifications
+
+CodeReady Analytics is an open-source analytics platform aimed primarily at assisting developers in getting insights and recommendations on the selected dependency/package stack used for developing their applications.
+
+This documentation is for developers who are interested in integrating their services or applications with CodeReady Analytics platform.
+
+## **Supported ecosystems**
+
+Below are the ecosystems that CodeReady Analytics currently supports.  We will be adding support for more ecosystems in the near future.
+| | |
+|-|-|
+|**Ecosystem**|**Manifest file**|
+| Java Maven | pom.xml |
+| Python Pypi | requirements.txt |
+| NodeJS | package.json |
+
+## **Prerequisites**
+
+Any new integration with the CodeReady Analytics platform will have to do that using an existing 3-scale API management gateway setup. Please reach out to us by raising an issue [here](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/issues)
+
```

Comment:
To integrate with the CodeReady Analytics platform, you need to use an existing 3-scale API management gateway. Create an issue[here](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/issues) to reach out to us for the API management gateway.